src/styles/main.scss

```diff
@@ -0,0 +1,61 @@
+@import './utils/variables';
+@import './utils/mixins';
+
+body {
+  margin: 0;
+  height: 100vh;
+
+  @include flex-center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+
+  padding: 10px;
+  gap: $main-gap;
+  width: ($main-size * 7) + ($main-gap * 6);
```

Comment:
The calculation of width should be placed in a mixin for better maintainability and readability, especially when dealing with a formula that includes variables and arithmetic operations.